### PR TITLE
Add build metadata and new GUID

### PR DESCRIPTION
**Pull request by joshuaboniface**
Adds a configuration for plugin build infrastructure (https://github.com/joshuaboniface/jellyfin-build-preview) and updates the GUID and versioning to match.

### CODE REVIEW

**Comment by cvium** [comment]
You forgot to update https://github.com/jellyfin/jellyfin-plugin-anime/blob/master/Jellyfin.Plugin.Anime/Configuration/configPage.html#L57

File: build.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+overview: >
+  Manage your Anime in Jellyfin. Supports several different metadata providers and options for
+  organizing your collection.
+
+  Requires Jellyfin 10.2.0 or newer.
+category: "General"
+artifacts:
+  - "Jellyfin.Plugin.Anime.dll"
+build_type: "dotnet"
+dotnet_runtime: "linux-x64"
```
Comment (JustAMan) [inline build.yaml]: umm, so it won't build for Windows?..

Comment (joshuaboniface) [reply build.yaml]: No, nor should it be - this is for build automation on a Linux machine to build the repo. I mean, sure one could build it on Windows using `dotnet publish`, or editing this and using `build.py`, but for this sort of infrastructure automation I don't see the point of sacrificing simplicity for supporting every possible platform.

Comment (JustAMan) [reply build.yaml]: I would argue that you'd need to use `dotnet publish` yourself to make sure all dependencies are there... given plugins can contain dependencies from nuget and stuff like that.

And I don't think that we should hardcode infrastructure platform in the metadata of a plugin, that is, if `build_type == dotnet` then build infra should specify whatever runtime is correct without it being explicitly put in the manifest.

Comment (joshuaboniface) [reply build.yaml]: That's a good point, I took a shortcut there to support the possibility of plugins being built with different options here, but they've all turned out to be the same. So I can probably move that logic out into the `build.py` script itself.

Comment (joshuaboniface) [reply build.yaml]: This is why I posted it for review ;-)

Comment (Matt07211) [reply build.yaml]: If I am reading the [docs](https://docs.microsoft.com/en-us/dotnet/core/rid-catalog) correctly, the only time you really should be specifying the runtime is when you are using platform specific assets/features. So I am of the opinion that you should be able to safely drop that for the anime plugin

Comment (joshuaboniface) [reply build.yaml]: You're right, I didn't need `--self-contained`, and hence didn't need this.

**Review by Bond-009 — DISMISSED** [review]

**Review by Bond-009 — APPROVED** [review]

**Pull request merged**